src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
> I'd say it's better to have additional warnings than suppressing everything.

:+1: here!

> Yes, i thought so, but couldn't reproduce it. Maybe it's fixed in the extension itself in the meantime.

If we can't reproduce this now, this means we should at least add a test this is our current assumption now (so we spot any regressions in the extension) :+1: